components/playlist/browser/resources/components/player.tsx

```diff
@@ -107,7 +107,11 @@ export default function Player () {
         videoElement.src !== currentItem?.mediaPath.url
       ) {
         videoElement.src = currentItem.mediaPath.url
-        videoElement.currentTime = currentItem.lastPlayedPosition
+        if (currentItem.lastPlayedPosition < +currentItem.duration / 1e6 - 5) {
```

Comment:
IMO, it would be more better for checking percentage instead of 5s as video could have short duration(ex, 8s). WDYT?

Reply:
Hum, On the other hand, when a video is long then the percentage could be up to several minutes. I think taking fixed seconds and making exceptions for short videos under 10 secs would be better.

Reply:
or maybe we can combine both condition with max?

Reply:
Well, I gave it a try but even for short videos having fixed 5s throttle feels more reasonable to me. Because I couldn't reconize video when resuming at 7 sec or 6 sec out of 8sec long video.

Reply:
Yeah, you're right. less than 5s is too short to recorgnize for all videos.